### ISSUE TITLE
Error Occurs When Running Program After Packaging with PyInstaller

### ISSUE THREAD

**Issue by bigsu** [pygeoops/pygeoops#108]
I encountered an error when running the program after packaging it with PyInstaller.
Python 3.11.9  `pip install pygeoops==0.5.0a1`
![image](https://github.com/user-attachments/assets/ff2769e1-fcaa-4149-b9ac-20bda32331f5)

**Comment by theroggy** [comment]
I've never used such a tool, but most likely it is not pygeoops that gives issues, as it is plain python, but shapely, which is partly C.

The error also clearly mentions that "No package metadata was found for shapely"... so I think you need to look in that direction. Based on a quick google you apparently need a "pyinstaller hook" file for packages like shapely and you should be able to find it in [pyinstaller-hooks-contrib](https://github.com/pyinstaller/pyinstaller-hooks-contrib).

**Comment by bigsu** [comment]
Yes, I am also confused. This error only occurs after packaging.It works fine in vscode
